frontend/src/selectors/staticContentSelectors.js

```diff
@@ -25,3 +25,24 @@ export const getMineTSFRequiredDocumentsHash = createSelector(
   (requiredDocuments) =>
     requiredDocuments.reduce((map, { value, label }) => ({ [value]: label, ...map }), {})
 );
+
+export const getConditionalDisturbanceOptionsHash = createSelector(
```

Comment:
Let's chat about how we can make this easier to reason about

Reply:
easier to reason about? 
easier to understand you mean?

Reply:
We'll address this in the commodity cleanup, as discussed IRL. Will mark as resolved